lib/iris/_lazy_data.py

```diff
@@ -104,15 +95,10 @@ def as_lazy_data(data, chunks=None, asarray=False):
         # but reduce it if larger than a default maximum size.
         chunks = _limited_shape(data.shape)
 
+    if isinstance(data, ma.core.MaskedConstant):
+        data = ma.masked_array(data.data, mask=data.mask)
```

Comment:
Is this related, or fixing a different problem ?

Reply:
This was the way the code was before the original change. I had moved it into the `_getall` function, which I've now removed. I _think_ it's necessary because dask doesn't like `MaskedConstant`.